### ISSUE TITLE
`http` protocol is hardcoded in the URL

### ISSUE THREAD

**Issue by DeadNews** [umputun/secrets#40]
## `http` protocol is hardcoded in the URL

https://github.com/umputun/secrets/blob/621b9aa93c6dedc0bcf20e6fdf2fee3cf58e2b74/app/server/web.go#L162

How to use `https` protocol instead of `http`?

## Steps To Reproduce

1. Open https://safesecret.info and click on «Generate Secure Link».
2. Get the link: `http://safesecret.info/message/{{ UUID }}`.

**Comment by umputun** [comment]
@oneils - i think this was a regression introduced by #31

could you pls take a look?

**Comment by DeadNews** [comment]
https://github.com/umputun/secrets/blob/621b9aa93c6dedc0bcf20e6fdf2fee3cf58e2b74/app/server/web.go#L162

Possible solutions are to replace the `domain` variable with `root_url` or to add a `protocol` variable.

Example of [grafana config](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#server) with these variables:
```ini
[server]
domain = grafana.example.com
enforce_domain = true
root_url = https://grafana.example.com
```

**Comment by oneils** [comment]
@umputun , sure. I'll take it.

**Comment by umputun** [comment]
fixed by #41
